### DISCUSSION TITLE
Cable management

### DISCUSSION THREAD

**Discussion by jhu18464** [phord/Jarvis#3] [Ideas]
Could you add some picture of the cable management between :
- Handset model: JCHT35M16-1
- Desk controller model: FullyCB2C-A
- ESP8266 wifi module
To have a better understanding of how it is cabled and to reproduce it at home 👍 
Thank you to share this project with us

**Comment by jhu18464** [comment]
Any thought on that @phord ? :)

**Comment by phord** [comment]
Sorry, I thought I answered you already, but it was a [separate discussion here: Issue #1](https://github.com/phord/Jarvis/issues/1#issuecomment-741101288) 

As for cable management, well... I just hook it up inline under the desk.  All of the jarvis cabling is DIY.  Here's a picture of the desk underside.  I think you can get the rest of the details from that issue link.
![PXL_20201211_182034502](https://user-images.githubusercontent.com/123908/102549862-b65d9300-4071-11eb-9af8-086ab18b64a9.jpg)
